### PR TITLE
Add some ad-hoc design tweaks for Actueel

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**
## Summary

Some of these things are "quick and dirty" because components will / should change in the next sprint.

### CODE REVIEW

File: packages/app/src/components-styled/warning-tile.tsx [hunk 1]
```diff
@@ -121,6 +121,10 @@ const Children = styled.div<{ variant: WarningMessageVariant }>(
           mb: 0,
         },
       },
+
+      '> a': {
+        color: 'hotpink !important',
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/components-styled/warning-tile.tsx:126]: Exactly this color?

File: packages/app/src/domain/topical/components/search/search-input.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export function SearchInput() {
 }
 
 export const paddedStyle = css({
-  p: ['1rem', null, null, '1.5rem'],
+  p: ['1rem', null, null, '1.0rem'],
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/domain/topical/components/search/search-input.tsx:56]:
```suggestion
  p: '1rem',
```

File: packages/app/src/locale/nl.json [hunk 3]
```diff
@@ -2256,11 +2242,7 @@
     "laatst_bijgewerkt": "Laatst bijgewerkt: {{date}}",
     "secties": {
       "artikelen": {
-        "titel": "Actuele artikelen",
-        "link": {
-          "text": "Bekijk alle artikelen ",
-          "href": "/artikelen"
-        }
+        "titel": "Actuele ontwikkelingen"
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/locale/nl.json:2245]: This won't work, lokalize willm put it back for you

File: packages/app/src/locale/nl.json [hunk 4]
```diff
@@ -2387,7 +2369,7 @@
       "selecteer_toelichting": "Iedere twee weken wordt bekeken of de situatie rond het coronavirus zich positief of negatief ontwikkelt. De laatste keer is dit op 2 maart gebeurd. Op welk niveau een regio zit, hangt af van het aantal positieve testen en het aantal ziekenhuisopnames. [Meer informatie over de risiconiveaus](/over-risiconiveaus).\n\n**De landelijke lockdown blijft van kracht tot en met ten minste 15 maart. [Bekijk hier de maatregelen van de lockdown](landelijk/maatregelen).**"
     },
     "risoconiveau_maatregelen": {
-      "title": "Risiconiveau & maatregelen",
+      "title": "Risiconiveau",
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/locale/nl.json:2372]: Should be done in lokalize

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/components-styled/warning-tile.tsx (hunk 1 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/components-styled/warning-tile.tsx:126]: Hehe foutje bedankt

File: packages/app/src/locale/nl.json (hunk 3 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/locale/nl.json:2245]: Ze zoeken het maar uit

File: packages/app/src/locale/nl.json (hunk 4 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/locale/nl.json:2372]: OK I'll try to remember that. They will find out.

**Review by VWSCoronaDashboard10 — APPROVED** [review]

**Pull request merged**